spring/boot2-webflux-autoconfigure/src/test/java/com/linecorp/armeria/spring/web/reactive/ArmeriaServerHttpResponseTest.java

```diff
@@ -171,6 +172,31 @@ void returnHeadersAndBody() throws Exception {
         await().until(() -> httpResponse.whenComplete().isDone());
     }
 
+    @Test
+    void ignoreCancelledSubscriptionException() throws Exception {
+        final ServiceRequestContext ctx = ServiceRequestContext.of(HttpRequest.of(HttpMethod.HEAD, "/"));
+
+        final CompletableFuture<HttpResponse> future = new CompletableFuture<>();
+        final ArmeriaServerHttpResponse response = response(ctx, future);
+
+        response.setStatusCode(HttpStatus.OK);
+        response.getHeaders().add("Armeria", "awesome");
+        assertThat(future.isDone()).isFalse();
```

Comment:
Could simplify with:
```suggestion
        assertThat(future).isNotDone();
```